recipes/icdiff/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "icdiff" %}
+{% set version = "1.9.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 66972dd03318da55280991db375d3ef6b66d948c67af96c1ebdb21587e86655e
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - icdiff=icdiff:start
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  commands:
+    - icdiff --help
+
+  imports:
+    - icdiff
+
+about:
+  home: http://www.jefftk.com/icdiff
+  license: Python 2.6.2
+  license_family: OTHER
+  # https://github.com/jeffkaufman/icdiff/issues/137
+  license_file:
```

Comment:
As the license is not being packaged in pypi, you can add a license file with this pull request copying this license
https://www.python.org/download/releases/2.6.2/license/

Reply:
Ahh right, forgot this was the procedure. Done!